boards/v2/qemu/qemu_leon3/Kconfig.qemu_leon3

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2019-2020 Cobham Gaisler AB
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_QEMU_LEON3
+	select SOC_LEON3
+	select QEMU_TARGET
+	select CPU_HAS_FPU
```

Comment:
those are still here, ref: https://github.com/zephyrproject-rtos/zephyr/pull/67475#discussion_r1453600423

Reply:
Removed this time